index.html

```diff
@@ -280,15 +282,17 @@
3 5.5 4.5 0 3.1-2.3 4.7-7 4.7h-2.1zm37.8 6.7h9l3.1 8.2h5.9l-10.9-28.8-.2-.5H631l-11.1 29.3h5.8l3-8.2zm4.6-13l2.6 7.8h-5.2l2.6-7.8zm-395.8 73.8h-36v-31.5h-8.2v71.3h8.2v-32.3h36v32.3h8.2v-71.3h-8.2v31.5zm44.2-14.2c-15.4.3-28 13-28 28.4 0 15.5 12.6 26.9 28.1 26.9 11.9 0 22.6-6.3 26.5-17.6l.2-.7h-8.3l-.1.3c-2 5.5-8.1 11.9-18.5 12.1-10 0-19.2-8.4-19.9-18.1h48.1v-.5c.4-6.4-1-12.1-4.1-17-3.7-6.6-12.1-13.7-24-13.8zm-19.9 24.2c2-11.2 11.5-17 20-17 8.1 0 17.8 5.3 20.1 17h-40.1zM364 172.5l-11.9-37.1h-6.7l-11.8 37.1-16.2-43.1-.1-.3h-8.2l20.4 52.3.1.3h8.4l10.7-33 10.9 33h8.4l20.6-52.7h-8.1L364 172.5zm30.7 9.2h7.8v-71.3h-7.8v71.3zm42.8-54c-15.4.3-28 13-28 28.4 0 15.5 12.6 26.9 28.1 26.9 11.9 0 22.6-6.3 26.5-17.6l.2-.7H456v.3c-2 5.5-8.1 11.9-18.5 12.1-10 0-19.2-8.4-19.9-18.1h48.1v-.5c.4-6.4-1-12.1-4.1-17-3.8-6.6-12.1-13.7-24.1-13.8zm-19.9 24.2c2-11.2 11.5-17 20-17 8.1 0 17.8 5.3 20.1 17h-40.1zm96.5-41.5h-7.8V129h-21.1v-18.7h-7.8V129h-11.1v7.2h11.1v45.5h7.8v-45.5h21.1v45.5h7.8v-45.5h12.7V129h-12.7v-18.6zm416.7 71.3h7.8v-54h-7.8v54zm-20.4-72.6h-7.8v18.7h-11.1v7.2h11.1v46.8h7.8V135h12.7v-7.2h-12.7v-18.7zm140.4 23.9c-4.6-4.6-9.9-6.6-17.1-6.6-7.6 0-13 3.6-15.9 7.3v-5.9h-7.8v54h7.8v-29.7c-.1-6 1.6-11.1 4.8-14.1 4.2-4.1 9.6-4.4 11.1-4.4s6.9.3 11.1 4.4c3.1 3 4.9 8.1 4.9 14.1v29.6h7.8v-29.6c0-8.2-2.3-14.8-6.7-19.1zm-120-10.7h7.8v-13.2h-7.8v13.2zm-53.5 13.8c-6.3-8.8-16.2-9.7-20.3-9.7-18.1 0-29 14.3-29 28.2 0 15.9 12.7 28.4 29 28.4 4 0 13.8-1 20.3-9.7v8.4h7.8v-54h-7.8v8.4zm-20.2 39.7c-12 0-21.1-9-21.1-20.9 0-14 10.6-21.3 21.1-21.3 11.4 0 20.6 9.5 20.6 21.2 0 11.8-9.1 21-20.6 21zm118-49.4c-13.5 0-28.1 11-28.1 28.7 0 15.1 12.9 27.9 28.1 27.9 15.3 0 28.3-13 28.3-28.3-.1-15.4-13-28.3-28.3-28.3zm.1 49.4c-11.2 0-20.4-9.4-20.4-20.8 0-13.2 10.6-21.4 20.4-21.4 8.2 0 20.4 6.6 20.4 20.8-.1 13.4-10.4 21.4-20.4 21.4zm-355-49.4c-13.5 0-28.1 11-28.1 28.7 0 15.1 12.9 27.9 28.1 27.9 15.3 0 28.3-13 28.3-28.3 0-15.4-13-28.3-28.3-28.3zm.1 49.4c-11.2 0-20.4-9.4-20.4-20.8 0-13.2 10.6-21.4 20.4-21.4 8.2 0 20.4 6.6 20.4 20.8-.1 13.4-10.4 21.4-20.4 21.4zm-70.3 5.9h8.2V149h28v-7.5h-28v-24.9h29v-7.5H550v72.6zM814.6 136c-6.8-8.9-17.2-9.6-20.3-9.6-18.1 0-29 14.3-29 28.2 0 15.9 12.7 28.4 29 28.4 6.4 0 14.9-2.6 20.3-9.7v8.4h7.8v-72.6h-7.8V136zm-20.2 39.8c-12 0-21.1-9-21.1-20.9 0-14 10.6-21.3 21.1-21.3 11.4 0 20.6 9.5 20.6 21.2 0 11.8-9.1 21-20.6 21zm-100.1-19.6c0 9.4-2.5 12.7-5.4 15.5-2.8 2.8-6.3 4.1-10.8 4.1-5 0-9-1.6-11.9-4.8-2.5-3.1-3.7-7.5-3.7-13.6v-29.6h-7.8v29.7c0 7.6 1.6 13.5 4.8 17.4 2.4 3 8.1 8.1 18.6 8.1 9.8 0 14.6-5.3 16.6-8.3v7.1h7.4v-54h-7.8v28.4zm41.1-29.8c-7.6 0-13 3.6-15.9 7.3v-5.9h-7.8v54h7.8v-29.7c-.1-6 1.6-11.1 4.8-14.1 4.2-4.1 9.6-4.4 11.1-4.4s6.9.3 11.1 4.4c3.1 3 4.9 8.1 4.9 14.1v29.6h7.8v-29.6c0-8.2-2.3-14.7-6.7-19-4.6-4.7-9.9-6.7-17.1-6.7z"/><path fill="#7CAF41" d="M0 125.4h6.8V0H0v125.4zm94-70.8H83.7V0H77v54.6H23.6V0h-6.8v61.4H77v10.2H17v53.7h6.8v-47H77v30.1h6.8v-30H94v30.1h6.8V.1H94v54.5zm0 17H83.7V61.4H94v10.2zm352.4.4v-7.6H441V72h-7.5v5.4h7.5V85h5.4v-7.6h7.5V72h-7.5z"/></svg>
```

Comment:
I ended up implementing contrast styles for the new Hewlett logo, since this is easy to do with SVG.

Is there any alternative to `<use>` if we want to style with CSS?